jplag/src/main/java/jplag/CLI.java

```diff
@@ -111,6 +108,8 @@ public JPlagOptions buildOptionsFromArguments(Namespace namespace) {
         options.setSubdirectoryName(namespace.getString("S"));
         options.setFileSuffixes(fileSuffixes);
         options.setExclusionFileName(namespace.getString("x"));
+        ComparisonMode.fromName(namespace.getString("c")).ifPresentOrElse(it -> options.setComparisonMode(it),
```

Comment:
Does this print the line every time one does not use the flag? Because this is probably a little bit verbose

Reply:
`Unknown comparison mode, using default` is only printed when explicitly using `-c x` with x ∉{normal, parallel}.